meta-balena-zeus/recipes-core/plymouth/plymouth_0.9.4.bbappend

```diff
@@ -0,0 +1,8 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = " \
+    file://0002-plymouth-systemd-append.patch \
+    file://0003-plymouth-Default-theme-is-resin.patch \
+    file://0004-Avoid-depending-on-systemd-ask-password-path-unit.patch \
+    file://0005-dont-start-services-in-container.patch \
+    "
```

Comment:
Can the plymouth patches be moved to meta-balena-common? They seem to be in all the yocto versions we support so might as well move them to the common area.

Reply:
Pulled plymouth into meta-balena-common